src/lib/Metadata/Interface.cpp

```diff
@@ -75,7 +75,8 @@ class TrancheBuilder
         std::vector<SymbolID>& M,
         const Info& I)
     {
-        M.emplace_back(I.id);
+        if(std::ranges::find(M, I.id) == M.end())
```

Comment:
Will this ensure we take the right inherited member, though?

Reply:
@alandefreitas All this does is ensure that elements in the tranche are unique.

Reply:
So let's at least open another issue